test/language/statements/with/unscopables-inc-dec.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2017 Mozilla Corporation. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description:
```

Comment:
can you add a `>` at the end of this line to make it compatible with monkeyYaml?

`description: >`

Ref https://github.com/tc39/test262/pull/862